src/ui/state/vault.rs

```diff
@@ -162,6 +163,19 @@ pub enum VaultSection {
 }
 
 impl VaultSection {
+    pub fn title(&self, vault: &model::Vault) -> &'static str {
+        match self {
+            Self::Unloaded => "Loading...",
+            Self::OnchainTransactions { .. } => match vault.status {
+                VaultStatus::Funded | VaultStatus::Unconfirmed => "Deposit detail",
+                _ => "Vault detail",
```

Comment:
nits:
- I would put "" when it's Unloaded, so you don't see the title changing (it doesn't take much to load anyway)
- I would use "details" instead of "detail"

Reply:
Not resolved: "Deposit details" R170 :)